docs/staking.md

```diff
@@ -0,0 +1,48 @@
+
+:::note
+
+In a Substrate-based network, time is divided in units called **eras**. In [HydraDX Snakenet](/snakenet), *1 Era = 24 hours*.
+
+:::
+
+Participating as a validator requires a certain level of technical knowledge for securely setting up and maintaining a validator node. Misbehavior of the validator node can be punished by slashing, resulting in an involuntary loss of funds for you and your nominators. If you believe that you have the necessary experience for running a validator node, you can refer to our [validator guide](/node_setup). Otherwise, we highly recommend that you consider participating as a nominator.
+
+### Nominators
+
+Nominators help secure the network by nominating validators to be elected in the active validator set. They do so by staking their HDX tokens with the validators of their choice. The nomination process does not require running and maintaining nodes, making this form of staking more accessible to everybody. Tokens which are used to nominate validators are *bonded*, meaning that they are frozen and cannot be used for other purposes. It is at any time possible to change or stop your nominations which will be reflected at the end of the current era. Nominators can also free up their tokens, however this will only become effective after a waiting period of *28 days* following the unbonding transaction.
+
+Before nominating, you should always do your due diligence and research the credibility of the chosen validators. You can do so by checking their identity as well as historical information like era points, elected stake, rewards and slashes. At the beginning of Snakenet it might be difficult to find all this information - if you are in doubt about the choice of validators, approach us in Discord and we will share with you our community list of trusted validators.
+
+Another point to consider when choosing a validator is the *reward commission percentage*. This represents the proportion of the rewards which will be paid out to the validator for providing its services to nominators. The lowest commission percentage is not always the best - running a performant and available node has high operational costs which can only be covered sustainably by demanding a realistic reward commission.
+
+:::note
+
+We strongly recommend that you spread your stake by **nominating several validators**. This enhances the decentralization of the network and increases your chances of electing a trustworthy node in the set of active validators.
```

Comment:
@jak-pan is the stake really spread across several validators? from my polkadot-experience it seemed like you just get elected by one validator.

Reply:
```suggestion
We strongly recommend to **nominate several validators**. This enhances the decentralization of the network and increases your chances of being elected by an validator from the active validator set.
```

Reply:
```suggestion
We strongly recommend to **nominate several validators**. This enhances the decentralization of the network and increases your chances of your stake being included in the elected validator set.
```